tide_landing_page.module

```diff
@@ -472,3 +474,53 @@ function _tide_landing_page_form_node_form_after_build(array $form, FormStateInt
 
   return $form;
 }
+
+/**
+ * Get all referenced nodes for a node in promotion & navigation card paragraph.
+ *
+ * @param \Drupal\node\NodeInterface $node
+ *   The node context.
+ *
+ * @return null|array
+ *   The node ids.
+ */
+function _tide_landing_page_get_all_ref_nodes(NodeInterface $node) {
+  try {
```

Comment:
please see the comment.

Reply:
hey @MdNadimHossain , IMO, the `try`  is unnecessary. 
as there is no IO operation happening from here, and drupal does these check or exceptions handling for use, especially for using Drupal storage query or `\Drupal::database()`.
in this case, if you want to catch an exception, I personal suggest that you could do this from 
`$result = $query->execute();` line,  the execute statement.